examples/gallery/plot/points-categorial.py

```diff
@@ -0,0 +1,48 @@
+"""
+Color points by categories
+---------------------------
+The :meth:`pygmt.Figure.plot` method can be used to plot symbols which are
+color-coded by categories.
+"""
+
+import numpy as np
+import pandas as pd
+import pygmt
+
+# Load sample iris data, and convert 'species' column to categorical dtype
+df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/iris.csv")
```

Comment:
Just to spice things up a little (and make it slightly different to the plot3d example), do you want to try the penguin datasst? It's almost a drop in replacement for iris. See https://seaborn.pydata.org/examples/multiple_regression.html

```suggestion
df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/penguins.csv")
```

Reply:
Good idea, modified the example 😉.